### ISSUE TITLE
Proto safe space testing

### ISSUE THREAD

**Issue by dberlow** [TypeNetwork/Amstelvar#75]
We have opsz and all of the y trans axes in, as well as a first cut at XOPQ min. 

I’d like to propose an instance at opsz max/XOPQ min where the XOPQ min is limited to a min at that corner of 96, instead of The full range of XOPQ which goes down to 36.
![d15947d4-f4b6-4d99-a207-e9e6625231c4](https://user-images.githubusercontent.com/1956494/39627481-10b6e690-4f74-11e8-94c7-44a74fdad145.png)

**Comment by dberlow** [comment]
I meant YOPQ in the previous comment.

**Comment by dberlow** [comment]
I meant, I meant XOPQ in the previous comment.

**Comment by sberlow** [comment]
@djrrb 
Is this complete?

**Comment by davelab6** [comment]
Sounds good DJR! 

For this project, Sam and I agreed that the 'safe space' is a v1 effort and he filed it there.

(I cleaned up the "I meant" comments for brevity)

**Comment by dberlow** [comment]
No idea where the "v" efforts  are described or where the "there" Sam filed it is. ;)

I have updated my Spreadsheets (https://docs.google.com/spreadsheets/d/1DgnZYL0PDEhF9hqlXrGWX5iEtNkEVq0rvCx9cun-Lzc/edit#gid=0) 

...for all the values of the parametric axes now complete.

I have located the instance of opsz max in the new parametric space and 
I am going to begin to "safen" all the space I can see, (the space not using YOPQ max) before DJR builds the vf again with the complete YOPQ max. 

I will be recording the safe values for each axis' combinations in my spread sheet and defining subsets of instances to thus restrict the design space. 

So, the issue is open and now started forward by me. 

All comments and suggestions are welcome, if I have not shared the spread sheet with you let me know, because in the end that is the only place I have to store this family data until the font is accurately doing so. 

Thanks.
